smarts/env/wrappers/format_obs.py

```diff
@@ -357,7 +378,12 @@ def get_spaces() -> Dict[str, Callable[[Any], gym.Space]]:
             "lane_width": gym.spaces.Box(low=0, high=1e10, shape=_WAYPOINT_SHP, dtype=np.float32),
             "pos": gym.spaces.Box(low=-1e10, high=1e10, shape=_WAYPOINT_SHP + (3,), dtype=np.float64),
             "speed_limit": gym.spaces.Box(low=0, high=1e10, shape=_WAYPOINT_SHP, dtype=np.float32),
-        })
+        }),
+        "signals": lambda _: gym.spaces.Dict({
+            "state": gym.spaces.Box(low=0, high=32, shape=(_SIGNALS_SHP,), dtype=np.int8),
+            "stop_point": gym.spaces.Box(low=-1e10, high=1e10, shape=(_SIGNALS_SHP, 2), dtype=np.float64),
```

Comment:
I could suggest making `_SIGNALS_SHP` a tuple since shape deals with dimensions rather than scalar.

```suggestion
            "state": gym.spaces.Box(low=0, high=32, shape=_SIGNALS_SHP, dtype=np.int8),
            "stop_point": gym.spaces.Box(low=-1e10, high=1e10, shape=_SIGNALS_SHP  + (2,), dtype=np.float64),
```

or

```suggestion
            "state": gym.spaces.Box(low=0, high=32, shape=_SIGNALS_SHP, dtype=np.int8),
            "stop_point": gym.spaces.Box(low=-1e10, high=1e10, shape=(_SIGNALS_SHP[0], 2), dtype=np.float64),
```